docs/Wdio_Utility.md

```diff
@@ -62,76 +61,54 @@ There are a few things to note about the webdriver.io configuration provided by
 
 - Test files should use `*-spec.js` naming format. The default spec search paths are `./tests/wdio/**/*-spec.js` and `./packages/*/tests/wdio/**/*-spec.js`.
 - Use `/test_url_path` to direct test urls. This is appended to the `baseUrl` provided in the config.
+- The TerraService automatically sets the `caret-color` CSS property to `transparent` at the root level whenever a page is loaded or refreshed, although this can be overridden by CSS further down the tree. If it is overridden you can change it back to `transparent` with `Terra.hideInputCaret()` (scroll down for documentation).
 
 Then, to assist with testing, the TerraService provides the Terra global helper to make testing easier:
```

Comment:
Can we break these into two sections:

1. Test Setup Helpers
2. Test Assertion Helpers

Reply:
Done in 04987d77e2d549eee0d8c03ad3f07fe0fa4103c2